src/yadism/observable_name.py

```diff
@@ -120,18 +105,13 @@ def is_heavy(self):
             is_heavy : bool
                 is a heavy flavor?
         """
-        return self.flavor != "light"
+        return "light" not in self.flavor
```

Comment:
Are we sure?

The difference from the previous one and this one is that `heavylight` was previously *heavy*, and now it is not any longer.

In principle, we might not need `heavylight` any longer. In which case it's just the same (but, if we're still using it, we should make sure of the effect of this).